doc/vim-go.txt

```diff
@@ -1419,6 +1419,16 @@ Supported values are "", "quickfix", and "locationlist". >
 
   let g:go_list_type = ""
 <
+
+                                                       *'g:go_list_autoclose'*
+
+Specifies whether the quickfix/location list should be closed automatically
+in the absence of errors.  The default value is 1.
+If you prefer to keep a long running error window open, you can disable 
+this by setting the value to 0.
```

Comment:
I think people using Vim already know that setting to `0` means disabling. So the whole text should be actually just like the others in the form of:

```
Specifies whether the quickfix/location list should be closed automatically in
the absence of errors. If you prefer to keep a long running error window open,
you can disable this by using this setting. By default it's enabled.
```